File: Pig_Latin/Pig_Latin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
string[] sen = Console.ReadLine().Split();
foreach(string x in sen)
 Console.Write(x.Substring(1)+x[0]+"ay ");
        }
    }
}

File: Pig_Latin/Pig_Latin.java
import java.util.Scanner;

public class Program
{
    public static void main(String[] args) {
  Scanner input = new Scanner(System.in);
  String [] sen = input.nextLine().split(" ");
  for(String x : sen)
  {
  System.out.print(x.substring(1)+x.charAt(0)+"ay ");
  
  }
}
}
